code/modules/mob/living/basic/vermin/mothroach.dm

```diff
@@ -65,12 +69,19 @@
 		playsound(loc, 'sound/voice/moth/scream_moth.ogg', 50, TRUE)
 
 /datum/ai_controller/basic_controller/mothroach
-	blackboard = list()
+	blackboard = list(
+		BB_FLEE_TARGETING_STRATEGY = /datum/targeting_strategy/basic,
+		BB_TARGETING_STRATEGY = /datum/targeting_strategy/basic/allow_items,
+	)
 
 	ai_traits = STOP_MOVING_WHEN_PULLED
 	ai_movement = /datum/ai_movement/basic_avoidance
 	idle_behavior = /datum/idle_behavior/idle_random_walk
 	planning_subtrees = list(
+		/datum/ai_planning_subtree/find_food,
```

Comment:
I've wanted mothroaches to eat clothes for a while so I am glad someone else did it.
You should give them a pretty long cooldown on doing so though, so as to not make them too annoying. They should only be eating clothing like once a minute, not every time they see some.

Reply:
(If we don't log mobs eating and thus destroying items then we probably should).

Reply:
theyre now logged